inference-engine/ie_bridges/python/CMakeLists.txt

```diff
@@ -76,6 +76,14 @@ endif()
 
 # install
 
+ie_cpack_add_component(${PYTHON_BRIDGE_CPACK_PATH})
+
+install(FILES requirements.txt
```

Comment:
So is this the same file as the one installed to the `PYTHON_VERSION` subdirectory (line 89)? (https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/ie_bridges/python/requirements.txt) Maybe we could remove one of them?

Reply:
yes, we will do it in the next interaction